### DISCUSSION TITLE
invalid sql statement SQL = / - when open toad scripts

### DISCUSSION THREAD

**Discussion by alansenairj** [dbeaver/dbeaver#34886] [Q&A] [answered]
### Description

It cannot execute a toad created script and gets error parsing information. 

### DBeaver Version

CE 24.1.0

### Operating System

Fedora 39

### Database and driver

![image](https://github.com/dbeaver/dbeaver/assets/20565821/fde36d1a-3422-434a-a735-9e256089d5c9)
default oracle driver

### Steps to reproduce

1 import sqls
2 open sqls
3 execute sql

### Additional context

![image](https://github.com/dbeaver/dbeaver/assets/20565821/9479ed16-b9ca-4c39-9833-aa69570c27c0)

![image](https://github.com/dbeaver/dbeaver/assets/20565821/483f3139-ee49-42d1-99cc-d70e1068e943)

**Comment by E1izabeth** [comment, marked as answer]
`/` is a TOAD-specific command to execute statement
As far as I understand, you have `/` at the end of each statement in your scripts. So, you can change Statements delimiter setting to `/` in preferences
![image](https://github.com/dbeaver/dbeaver/assets/28875055/157df76b-f9ec-4fde-a5ff-db641974ee86)

Please let me know if it helps.
